cirq-core/cirq/transformers/merge_single_qubit_gates.py

```diff
@@ -47,11 +47,12 @@ def merge_single_qubit_gates_to_phased_x_and_z(
     """
 
     def rewriter(op: 'cirq.CircuitOperation') -> 'cirq.OP_TREE':
+        u = protocols.unitary(op)
+        if protocols.num_qubits(op) < 1:
```

Comment:
```suggestion
        if protocols.num_qubits(op) == 0:
```

Or is there a scenario `num_qubits()` would be negative and it would be meaningful to take this branch?